_docs/deprecated_and_archived_modules.md

```diff
@@ -0,0 +1,17 @@
+---
+layout: post
+title: Deprecated and Archived Modules
+date: 2019-11-29
+summary: Vox Pupuli policy on deprecating and archiving modules
+---
+
+## Archived Modules
+
+It is Vox Pupuli's mission to provide a home for modules so that development can continue after the original authors/maintainers have moved on.
+In rare circumstances a module may be archived and placed into a read-only state.  This typically will only occur if the module/project no longer serves a valid
```

Comment:
Rare or not it not the focus. We can archive a module as often as required. A simple `But` LGTM.

Reply:
I strongly believe it should be a last resort (and thus rare).  It's not fair on those who have migrated modules to us to archive them without *very* good reason.